packages/shared/src/hooks/streaks/useStreakRecover.ts

```diff
@@ -78,6 +78,22 @@ export const useStreakRecover = ({
     });
   }, [updateAlerts]);
 
+  const { data, isLoading } = useQuery<StreakQueryData>({
```

Comment:
Maybe a more failsafe approach would be to convert this to a const, and pass it to this hook as `enabled`?
https://github.com/dailydotdev/apps/blob/b65cd5575b653380bfa9927d6e93851f4ffbb7fe/packages/shared/src/components/modals/streaks/StreakRecoverModal.tsx#L174-L182

You can then just early return the hook, or set the query to be disabled?

Reply:
`recover` comes from query but adding `user` and `isStreaksEnabled` would be a great addition! Trying it :)